### PR TITLE
Docs: Document edge case for not completing a queued withdrawal for EigenPod

### PR DESCRIPTION
**Pull request by ypatil12**

### CODE REVIEW

File: docs/core/EigenPodManager.md [hunk 1]
```diff
@@ -434,13 +434,16 @@ Whether each withdrawal is a full or partial withdrawal is determined by the val
...
+    3. Complete their withdrawal (via [`DelegationManager.completeQueuedWithdrawal`](./DelegationManager.md#completequeuedwithdrawal)).
+
+If the Pod Owner only completes steps 1 and 2 above, the ETH of the pod owner is still staked through EigenLayer and can be used to service AVSs, even though their ETH has been withdrawn from the beacon chain. The protocol allows for this edge case. 
 
 *Beacon chain proofs used*:
 * [`verifyStateRootAgainstLatestBlockRoot`](./proofs/BeaconChainProofs.md#beaconchainproofsverifystaterootagainstlatestblockroot)
 * [`verifyWithdrawal`](./proofs/BeaconChainProofs.md#beaconchainproofsverifywithdrawal)
 * [`verifyValidatorFields`](./proofs/BeaconChainProofs.md#beaconchainproofsverifyvalidatorfields)
 
+An edge
```
Comment (ChaoticWalrus) [inline docs/core/EigenPodManager.md:446]: seems to be errant / added on accident?

File: docs/core/EigenPodManager.md [hunk 2]
```diff
@@ -434,13 +434,16 @@ Whether each withdrawal is a full or partial withdrawal is determined by the val
 * Full withdrawals are performed when a Pod Owner decides to fully exit a validator from the beacon chain. To do this, the Pod Owner should follow these steps: 
     1. Undelegate or queue a withdrawal (via the `DelegationManager`: ["Undelegating and Withdrawing"](./DelegationManager.md#undelegating-and-withdrawing))
     2. Exit their validator from the beacon chain and provide a proof to this method
-    3. Complete their withdrawal (via [`DelegationManager.completeQueuedWithdrawal`](./DelegationManager.md#completequeuedwithdrawal))
+    3. Complete their withdrawal (via [`DelegationManager.completeQueuedWithdrawal`](./DelegationManager.md#completequeuedwithdrawal)).
+
+If the Pod Owner only completes steps 1 and 2 above, the ETH of the pod owner is still staked through EigenLayer and can be used to service AVSs, even though their ETH has been withdrawn from the beacon chain. The protocol allows for this edge case.
```
Comment (ChaoticWalrus) [inline docs/core/EigenPodManager.md:439]: I realized this is basically about just completing step (2) -- step (1) doesn't matter for the edge case, right? and they could always "complete the withdrawal as shares" to keep these (re)staked ETH in EigenLayer

**Review by ChaoticWalrus — APPROVED** [review]
Good change, had a couple minor nits, please at least clean up the errant 'an edge' before merging.

File: docs/core/EigenPodManager.md (hunk 2 above)
Comment (ypatil12) [reply docs/core/EigenPodManager.md:439]: Ah yes, good catch - updated.

**Pull request merged**